docs/ldml/tr35.md

```diff
@@ -2861,38 +2861,52 @@ Element content whose display may be affected in this way should include an expl
-| `charName`     | <pre>= s [A-Za-z0-9] [-A-Za-z0-9_\x20]* s</pre>                | TIBETAN LETTER -A                       |
-| `bracketedHex` | <pre>= '{' s hexCodePoint (s hexCodePoint)* s '}'</pre>        | \{61 2019 62}                           |
-| `hexCodePoint` | <pre>= hex{1,5} \| '10' hex{4}</pre>                           |                                         |
-| `hex`          | <pre>= [0-9A-Fa-f]</pre>                                       |                                         |
-| `s`            | <pre>= [:Pattern_White_Space:]*</pre>                          | optional whitespace                     |
-
-Some constraints on UnicodeSet syntax are not captured by this EBNF. Notably, property names and values are restricted to those supported by the implementation, and have additional constraints imposed by [[UAX44](https://www.unicode.org/reports/tr41/#UAX44)]. In addition, quoted values that resolve to more than one code point are disallowed in ranges of the form `char '-' char`.
+| `root`         | <pre>= prop<br/>\| '\[' '^'? s '-'? s seq\* \[\\$ \\-\]? s '\]' <br/>\| var</pre> | \\p\{x=y\},<br/>[abc],<br/>$myset                     |
+| `seq`          | <pre>= root \(s \[\\&\\\-\] s root\)\* s<br/>\| range s</pre>        | \[abc\]\-\[cde\], a                          |
+| `range`        | <pre>= element \('\-' element\)?       | a, a\-c, \{abc\}, \{ax\}\-\{bz\}                          | <!-- TODO: Support for string ranges of length != 1? -->
+| `prop`         | <pre>= '\\' \[pP\] '\{' propName \(\[≠=\] s value1\+\)? '\}'<br/>\| '\[:' '^'? propName \(\[≠=\] s value2\+\)? ':\]'</pre> | \\p\{x=y\}, \[:x=y:\]<br/> |
+| `propName`     | <pre>= s \[A\-Za\-z0\-9\] \[A\-Za\-z0\-9\_\\x20\]\* s</pre>                | General\_Category,<br/>General Category  |
+| `value1`       | <pre>= \[^\\\}\]<br/>\| '\\' quoted</pre>                         | Lm,<br/>\\n,<br/>\\\}                    |
+| `value2`       | <pre>= \[^:\]<br/>\| '\\' quoted</pre>                           | Lm,<br/>\\n,<br/>\\:                    |
+| `element`      | <pre>= char \| string \| var </pre> | %, b, \{hello\}, \{\}, \\x\{61 62\} |
+| `string`      | <pre>= '\{' \(s charInString\)\* s '\}' </pre> | \{hello\} |
+| `char`         | <pre>= \[^ \\^ \\& \\\- \\\[ \\\] \\\\ \\\{ \\$ \[:Pat_WS:\]\]<br/>\| '\\' quoted</pre> | a, b, c, \\n, \\\{, \\$ |
+| `charInString` | <pre>= \[^ \\\\ \\\} \[:Pat_WS:\]\]<br/>\| '\\' quoted</pre> | a, b, c, \\n, \{, $ <br/> *note: only quoted items that resolve to one code point are allowed* | <!-- TODO: Multi escape semantics? If flattening semantics, this note is unnecessary -->
+| `quoted`       | <pre>= 'u' \(hex\{4\} \| bracketedHex\)<br/>\| 'x' \(hex\{2\} \| bracketedHex\)<br/>\| 'U00' \('0' hex\{5\} \| '10' hex\{4\}\)<br/>\| 'N\{' charName '\}'<br/>\| \[\[\\u0000\-\\U00010FFFF\]\-\[uxUN\]\]</pre> | n, U0000FFFE, \{, $, \] <br/> _note: lengths are exact_ |
```

Comment:
We are in an already-escaped context here, so `n` is a valid production of `quoted`. Yes, it will be used as `\n`, because `quoted` always appears after a `\` , but that's not actually a concern of `quoted` itself.

Reply:
yup, my oversight.